github_pipeline_observability/change.finished.md

```diff
@@ -0,0 +1,35 @@
+### Change event for Semantic Dictionary
+
+`task.change.[open/merged/closed]`
+
+| Property | Value | Description | 
+|---|---|---|
+| event.id  |   |
+| event.kind  | `SDLC_EVENT`  | x 
+| event.version | `0.1.0` |  x
+| event.category | `task` |  x
+| event.type | `change` |  x
+| event.provider | `github.com` |  x
+| event.status |  [`merged`, `closed`, `open`] |  x
+| duration | *merged_at - created_at* |  x
+| start_time | *created_at* |  x
+| end_time | *merged_at or closed_at* |  x
+| timestamp | |  x
+| | | |
+| task.id | `871023010` | The unique task identifier.
+| task.name | `docs: Update of Readme.md` | A human-readable name for a task.
+| task.outcome | `success` | Outcome of a task run.
+| | | |
+| vcs.change.id  | `4711` | The identifier of the change, for example, pull request ID or merge request ID. It is typically unique per repository and generated by the version control system itself.
+| vcs.change.title  | `docs: Update of Readme.md`  | The human-readable title of the change, for example, pull request title or merge request title.  **Attention:** Ident to: "task.name"
+| vcs.change.state  | [`merged`, `closed`, `open`]  | The state of the change (pull request/merge request/changelist). **Attention:** Ident to: "event.status"
+| vcs.repository.name  | `johannes-b/platform-engineering-demo`  | The human readable name of the repository.
+| vcs.repository.url.full  | `https://github.com/johannes-b/platform-engineering-demo` | The repository's full URL.
+| vcs.ref.head.name  | `feature/34/add-button`  | The name of the reference in the repository. This can be a branch name or a tag name.
+| vcs.ref.head.revision  | `a8a375bec0beddd0d8c58fdfad4bf057d7f5fe88` | The revision in the repository. For Git, this is a synonym for a commit hash, whereas in SVN, it is a revision number.
+| vcs.ref.base.name  | `main` | The name of the reference in the repository. This can be a branch name or a tag name.
+| vcs.ref.base.revision  | `05a7199180e7df5f38c9c78efcf0e4474eb52065` | The revision in the repository. For Git this is a synonym for a commit hash, whereas in SVN it is a revision number.
+| | | |
+| **NEW** vcs.change.number  | `40`  | A human-readable number for the pull request or merge request. 
```

Comment:
I would propose the two properties: `vcs.change.numer` and `vcs.change.url.full`. `vcs.change.url.full` should point to the url of the PR, while `vcs.change.number` represents a human-readable number of the PR. GitHub differentiates between an ID and number; see: 

```{
  "action": "opened",
  "number": 58,
  "pull_request": {
    "url": "https://api.github.com/repos/johannes-b/platform-engineering-demo/pulls/58",
    "id": 2318069049,
    "node_id": "PR_kwDONk-fDM6KKu05",
    "html_url": "https://github.com/johannes-b/platform-engineering-demo/pull/58",
   ...
```